pkg/server/ca/manager.go

```diff
@@ -220,7 +226,7 @@ func (m *Manager) prepareX509CA(ctx context.Context, slot *x509CASlot) (err erro
 		return err
 	}
 
-	if err := m.appendBundle(ctx, trustBundle, nil); err != nil {
+	if err := m.AppendBundle(ctx, trustBundle, nil); err != nil {
```

Comment:
I have exported the CA Manager's appendBundle function because I needed to append a JWK to the bundle from the Node API handler (github.com/spiffe/spire/pkg/server/endpoints/node/handler.go). My first thought was to use the Datastore plugin directly, but then I realized that the CA Manager does some notification stuff that I wouldn't get triggered if I did that. So, it seemed to me that exporting the function was the way to go. I'm open to suggestion on how to improve (or reimplement) this, and happy to make all the necessary changes.

Reply:
if we implement a new method to do all validations as suggested before, we can avoid calling append from outside, and hava validations in a single place

Reply:
Yep, this is included on my next commit that I will push soon.